javascript-modal/main.js

```diff
@@ -0,0 +1,14 @@
+var $modalBtn = document.querySelector('.modal-btn');
+var $modal = document.querySelector('.modal');
+var $closeBtn = document.querySelector('.red-btn');
+
+$modalBtn.addEventListener('click', handleClick);
+$closeBtn.addEventListener('click', handleClick2);
+
+function handleClick(event) {
+  $modal.style.display = 'block';
+}
+
+function handleClick2(event) {
+  $modal.style.display = 'none';
+}
```

Comment:
You shouldn't set styling directly on DOM elements like this, this will become inline styling for the element, which makes for a mess if you ever want to change or overwrite this styling. You should make classes in a CSS stylesheet and just add or remove those classes using JavaScript.